src/python/nimbusml/examples/IidSpikeDetector.py

```diff
@@ -0,0 +1,38 @@
+###############################################################################
+# IidSpikeDetector
+from nimbusml import Pipeline, FileDataStream
+from nimbusml.datasets import get_dataset
+from nimbusml.time_series import IidSpikeDetector
+from nimbusml.preprocessing.schema import TypeConverter
```

Comment:
no need for this

Reply:
I forgot to remove the `TypeConverter` import when I added the implicit pre-transform to the `IidSpikeDetector`. Thanks for catching that.

I am unable to remove the other imports without breaking the execution of the example.